src/main/java/fiji/plugin/trackmate/action/CloseGapsByLinearInterpolationAction.java

```diff
@@ -0,0 +1,157 @@
+package fiji.plugin.trackmate.action;
+
+import fiji.plugin.trackmate.Model;
+import fiji.plugin.trackmate.Spot;
+import fiji.plugin.trackmate.SpotCollection;
+import fiji.plugin.trackmate.TrackMate;
+import fiji.plugin.trackmate.gui.TrackMateGUIController;
+import fiji.plugin.trackmate.gui.TrackMateWizard;
+import ij.gui.WaitForUserDialog;
+import net.imglib2.RealPoint;
+import org.scijava.plugin.Plugin;
+
+import javax.swing.*;
+import java.util.Set;
+
+/**
+ * This action allows to close gaps in tracks by creating new intermediate spots which are located at interpolated positions.
+ * This is useful if you want to measure signal intensity changing during time, even if the spot is not visible. Thus, trackmate
+ * is utilisable for Fluorescence Recovery after Photobleaching (FRAP) analysis.
+ *
+ * Author: Robert Haase, Scientific Computing Facility, MPI-CBG, rhaase@mpi-cbg.de
+ *
+ * Date: June 2016
+ *
+ */
+public class CloseGapsByLinearInterpolationAction extends AbstractTMAction {
```

Comment:
Could you reformat the code to comply to imglib2 style, using the imglib2 formatter if you use Eclipse?
It is there https://github.com/imglib/imglib2/blob/master/doc/imglib2-eclipse-mars-clean-up-formatter-style.xml